specs/pagination.spec.ts

```diff
@@ -0,0 +1,92 @@
+describe('InstantSearch - Page navigation', () => {
+  it('navigates to the e-commerce demo', async () => {
+    await browser.url('examples/e-commerce/');
+  });
+
+  it('navigates to next page', async () => {
+    await browser.setNextPage();
+  });
+
+  it('must have the expected results for page 2', async () => {
+    const hitsTitles = await browser.getHitsTitles();
+
+    expect(hitsTitles).toEqual([
```

Comment:
I wish we would have access to the the demo here, so we could show the absolute position in the UI & assert on it :(